Houseclub/src/main/java/me/grishka/houseclub/fragments/InviteListFragment.java

```diff
@@ -0,0 +1,348 @@
+
+        @NonNull
+        @Override
+        public IviteViewHolder onCreateViewHolder(@NonNull ViewGroup parent, int viewType){
+            return new IviteViewHolder();
+        }
+
+        @Override
+        public void onBindViewHolder(@NonNull IviteViewHolder holder, int position){
+            holder.bind(data.get(position));
+        }
+
+        @Override
+        public int getItemCount(){
+            return data.size();
+        }
+
+        @Override
+        public int getImageCountForItem(int position){
+            return data.get(position).photoUrl!=null ? 1 : 0;
+        }
+
+        @Override
+        public String getImageURL(int position, int image){
+            return data.get(position).photoUrl;
+        }
+    }
+
+    private class IviteViewHolder extends BindableViewHolder<FullUser> implements ImageLoaderViewHolder, UsableRecyclerView.Clickable{
+
+        public TextView name, bio;
+        public Button followBtn;
+        public ImageView photo;
+        private Drawable placeholder=new ColorDrawable(0xFF808080);
+
+        public IviteViewHolder(){
+            super(getActivity(), R.layout.user_list_row);
+
+            name=findViewById(R.id.name);
+            bio=findViewById(R.id.bio);
+            followBtn=findViewById(R.id.follow_btn);
+            photo=findViewById(R.id.photo);
+        }
+
+        @Override
+        public void onBind(FullUser item){
+            name.setText(item.name);
+            bio.setText(item.bio);
+            followBtn.setVisibility(View.GONE);
+            photo.setVisibility(View.GONE);
+        }
+
+        @Override
+        public void setImage(int index, Bitmap bitmap){
+            photo.setImageBitmap(bitmap);
+        }
+
+        @Override
+        public void clearImage(int index){
+            photo.setImageDrawable(placeholder);
+        }
+
+        @Override
+        public void onClick(){
+
+			AlertDialog.Builder builder = new AlertDialog.Builder(getContext());
+			builder.setTitle(R.string.invite_dialog_title);
+			builder.setMessage(getString(R.string.invite_dialog_text, item.name, item.dsplayname));
+
+			builder.setPositiveButton(R.string.button_positive, new DialogInterface.OnClickListener() {
+				@Override
+				public void onClick(DialogInterface dialog, int which) {
+					new InviteToApp(item.name, item.dsplayname, "")
+							.wrapProgress(getActivity())
+
+							.setCallback(new Callback<BaseResponse>(){
+								@Override
+								public void onSuccess(BaseResponse result){
+									Toast.makeText(getContext(), R.string.invite_ok, Toast.LENGTH_SHORT).show();
+								}
+
+								@Override
+								public void onError(ErrorResponse error){
+                                    Toast.makeText(getContext(), getString(R.string.invite_err, ""), Toast.LENGTH_SHORT).show();
+                                    error.showToast(getContext());
```

Comment:
This is indented with spaces for some reason

Reply:
fixed